### DISCUSSION TITLE
scribble diffusion result error

### DISCUSSION THREAD

**Discussion by ahneunhee** [community/community#121896] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi~

It's still not resolved and I'm stressed and sad. Many sites and blogs say that you can draw without logging in, but when you actually draw, no results are displayed, only hints about loading. I don't know how to solve it. Can you give a more detailed answer?

https://github.com/community/community/assets/168616181/7808571b-9904-417e-be1b-833fc7b9ee15

**Comment by ebndev** [comment]
Hi @ahneunhee,

Thanks for being a part of the GitHub Community, we're glad you're here!

It looks like you're looking for support from Scribble. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
